### PR TITLE
Disable unit tests for Windows e2e

### PR DESCRIPTION
**Pull request by jingxu97**
disable unit tests when running e2e tests for Windows to avoid it
blocking e2e tests

### CODE REVIEW

**Comment by jingxu97** [comment]
cc @pjh

File: config/jobs/kubernetes/sig-windows/windows-gce.yaml [hunk 1]
```diff
@@ -579,6 +579,8 @@ presubmits:
           value: "win2019"
         - name: PREPULL_YAML
           value: "prepull-head.yaml"
+        - name: KUBE_RELEASE_RUN_TESTS
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-windows/windows-gce.yaml:582]: please stop running this with `--build=release` in presubmit, it's extremely expensive and slow:  https://github.com/kubernetes/test-infra/issues/18943

**Comment by BenTheElder** [comment]
This now uses quick release anyhow. https://github.com/kubernetes/test-infra/pull/21068

File: config/jobs/kubernetes/sig-windows/windows-gce.yaml [hunk 2]
```diff
@@ -581,6 +581,10 @@ presubmits:
           value: "win2019"
         - name: PREPULL_YAML
           value: "prepull-head.yaml"
+        # Disable unit test for Windows for now.
+        # TODO: Should remove it after unit tests are fixed.
```
Comment (pjh) [inline config/jobs/kubernetes/sig-windows/windows-gce.yaml:585]: What is broken about the unit tests? Is there an issue tracking this? Will they ever be fixed, or will this never work for Windows?

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-windows/windows-gce.yaml:585]: 1. they're broken on windows, there is an issue in k/k
2. when running e2e we should not be running the unit tests
3. these run when building a release by default. they do not when running a quick release

**Comment by BenTheElder** [comment]
this PR should be unnecessary now https://github.com/kubernetes/test-infra/pull/21068

**Comment by dchen1107** [comment]
@BenTheElder Thanks!

**Pull request closed without merging**